### PR TITLE
[Translation] German

### PR DESCRIPTION
**Pull request by Intevel**
Hello,
I am not completely finished yet, the following translations are missing:

- [ ] /integrations/
- [ ] /announcements/2.understanding-how-fetch-works-in-nuxt-2-12.md/
- [ ] /docs/
- [ ] /tutorials/
 
 I don't think I forgot anything, otherwise everything should be translated. i'm working on the rest of the translation.

### CODE REVIEW

**Comment by Intevel** [comment]
Hey @smarroufin , of course, I'll keep working on the rest of the translation, is there any way to be notified when you can translate new stuff again? And I thought that I had translated `i18n/de-DE.js` completely, I will fix that right now.

**Comment by Intevel** [comment]
I updated it

**Comment by Intevel** [comment]
Are the translations marked only when they are complete or before? Because in the i18n are many files, but on nuxtjs.org are only 3 languages current.

**Comment by Intevel** [comment]
okay and when geht it merged? @smarroufin

File: content/de/announcements/1.going-full-static.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+Auch die Erfahrung der Entwickler ist nicht optimal:
+
+- Sie haben Zugriff auf `req` oder `res` im SSR, aber nicht, wenn Sie `nuxt generate` ausführen
+- `process.static` ist nur dann `true`, wenn `nuxt generate` ausgeführt wird, was die Entwicklung von Nuxt-Modulen oder Plugins für die Static Generierung verlangsamt
+- Sie müssen alle Ihre [Dynamic Routes](/docs/features/file-system-routing#dynamic-routes) in `generate.routes` angeben, was es schwieriger macht, da Sie dort keinen Zugriff auf nuxt-Module haben.
+- [SPA fallback](/docs/concepts/static-site-generation#spa-fallback) kann in der Entwicklung nicht getestet werden, der Fallback ist eine reine Client-Version Ihrer Nuxt-Anwendung, die beim Aufrufen einer 404-Seite geladen wird
+- `nuxt generate` führt standardmäßig `nuxt build` aus, wodurch die Erstellung der Website langsamer wird, wenn sich nur der Inhalt ändert
+
+Beachten Sie, dass es möglich war, vollen Static Support mit dem [nuxt-payload-extractor](https://github.com/DreaMinder/nuxt-payload-extractor) Modul zu haben, aber es war ausführlicher zu benutzen und hatte Einschränkungen.
```
Comment (manniL) [inline content/de/announcements/1.going-full-static.md:84]: "ausführlicher zu benutzten" -> Komische Übersetzung

**Review by manniL — CHANGES_REQUESTED** [review]
First of all - thanks for your work!

I started the review on a quite fine-grained level. While most of the content (which I've read so far) seems fine, some translations sound too artificial and sometimes there are inconsistencies as well as a mix of German and English.

Please take a look at my comments and review your translated files with regards to the mentioned issues. Then, I'll have another closer look ☺️ 

Nevertheless - great job so far!

File: content/de/announcements/0.nuxt-static-improvements.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+Wenn `nuxt generate` anschließend ausgeführt wird, erstellt es wieder eine Prüfsumme auf der Grundlage Ihrer Projektdateien und vergleicht sie dann mit der vorhandenen Prüfsumme in `node_modules/.cache/nuxt/build.json`.
+
+Wenn sie übereinstimmen, bedeutet das, dass sich nichts geändert hat, was neu erstellt werden muss, so dass wir direkt mit dem Rendern der Seiten beginnen können.
+
+Wird eine Nichtübereinstimmung festgestellt, bedeutet dies, dass ein vollständiger Rebuild erforderlich ist. Sie können auch in der Konsole nachsehen, welche Datei den Rebuild verursacht hat. Nach dem Build wird nuxt generate die neue Prüfsumme in `.nuxt/build.json` speichern. Sie können die vollständige Implementierung [hier überprüfen](https://github.com/nuxt/nuxt.js/pull/7712).
+
+
+### Zurück zu den alten Befehlen
+
+Mit Nuxt v2.13 haben wir `nuxt export` und `nuxt serve` eingeführt, die speziell für Static Targets entwickelt wurden. Mit Nuxt v2.14 sind sie veraltet, da `nuxt generate` und `nuxt start` das Ziel intelligent erkennen und builden.
```
[the remark this answers is not in the document]
Comment (Intevel) [reply content/de/announcements/0.nuxt-static-improvements.md:152]: Ähnlich wie der Kommentar darüber, ich finde kein vergleichbares Wort dazu weswegen ich über all builden bzw. Build geschrieben habe.

File: content/de/announcements/1.going-full-static.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+## Geschichte
+
+Nuxt hatte die Static Generierungsfunktion `nuxt generate` seit [v0.3.2](https://github.com/nuxt/nuxt.js/releases/tag/v0.3.2) (November 2016), seither haben wir sie auf verschiedene Weise verbessert, aber nie eine vollständige Static Generierung erreicht. Heute freue ich mich, ankündigen zu können, dass der vollständige Static Export nun mit Nuxt 2.13 möglich ist.
+
+## Aktuelle Probleme
+
+`nuxt generate` ist größtenteils Pre-Rendering, wenn Sie clientseitig navigieren, werden "asyncData" und "fetch" aufgerufen, _die eine Anfrage an Ihre API senden_. Viele Benutzer haben darum gebeten, einen "vollständigen Static" Modus zu unterstützen, was bedeutet, dass diese 2 Hooks bei der Navigation nicht aufgerufen werden, da die nächste Seite bereits vorgerendert wurde.
+
+Auch die Erfahrung der Entwickler ist nicht optimal:
```
[the remark this answers is not in the document]
Comment (Intevel) [reply content/de/announcements/1.going-full-static.md:76]: DX = Developer Experience
Wie soll ich das übersetzten? Experience bedeutet übersetzt ua. Erfahrung, Erlebnis klingt nicht so gut.

[the remark this answers is not in the document]
Comment (manniL) [reply content/de/announcements/1.going-full-static.md:76]: Einfach so lassen, ähnlich wie "User Experience"

File: content/de/announcements/1.going-full-static.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+<p>
+
+Notiz: In diesem Video benutzen wir `nuxt export` welches veraltet ist und durch `nuxt generate` ersetzt wurde.
+
+</p>
+
+## Inhaltsverzeichnis
+
+- [Zusammenfasung](#too-long-to-read)
```
Comment (marvin-j97) [inline content/de/announcements/1.going-full-static.md:54]:
```suggestion
- [Zusammenfassung](#too-long-to-read)
```

File: content/de/announcements/0.nuxt-static-improvements.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+
+Jetzt müssen Sie nur noch Ihren Build-Befehl von `nuxt build && nuxt export` in `nuxt generate` auf der von Ihnen verwendeten Plattform ändern. Wenn Sie ein CI verwenden, stellen Sie sicher, dass Sie `node_modules` richtig zwischenspeichern.
+
+### Dateien aus dem Cache ausschließen
+
+Standardmäßig ignoriert Nuxt diese Verzeichnisse, so dass bei Änderungen innerhalb dieser Verzeichnisse kein Build ausgelöst wird:
+
+- Build Ordner (`.nuxt/`)
+- Static Ordner (`static/`)
+- Generate dist (`dist/`)
```
[the remark this answers is not in the document]
Comment (Intevel) [reply content/de/announcements/0.nuxt-static-improvements.md:109]: @manniL

File: content/de/announcements/1.going-full-static.md [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+  - `nuxt start`: Bedienen sie ihre Anwendung vom Verzeichnis `dist/`
+
+### Was ist als nächstes zu tun?
+
+::alert{type="next"}
+Um mehr darüber zu erfahren, wie man von @nuxtjs/dotenv zur Laufzeitkonfiguration wechselt, lesen Sie [diesen Artikel](/tutorials/moving-from-nuxtjs-dotenv-to-runtime-config).
+::
+
+::alert{type="next"}
+[Newsletter abonieren](#subscribe-to-newsletter) um die nächsten Artikel und Ressourcen nicht zu verpassen.
```
Comment (marvin-j97) [inline content/de/announcements/1.going-full-static.md:175]:
```suggestion
[Newsletter abonnieren](#subscribe-to-newsletter) um die nächsten Artikel und Ressourcen nicht zu verpassen.
```

File: content/de/examples/8.middlewares/2.named.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Named Middleware
+title: Bennante Middleware
```
Comment (marvin-j97) [inline content/de/examples/8.middlewares/2.named.md:2]:
```suggestion
title: Benannte Middleware
```

File: content/de/examples/7.miscellaneous/2.lazy-loading-components.md [hunk 8]
```diff
@@ -1,10 +1,10 @@
 ---
-title: Lazy Loading Components
+title: Lazy-Loading-Komponenten
 description: Verwenden Sie Fetch in Ihren Komponenten, um Daten von einer API abzurufen, sowie für den automatischen Import und das langsame Laden von Komponenten.
```
Comment (marvin-j97) [inline content/de/examples/7.miscellaneous/2.lazy-loading-components.md:3]: Lazy != langsam

**Review by danimajo — APPROVED** [review]
LGTM

**Review by danimajo — APPROVED** [review]

**Comment by smarroufin** [comment]
Discussed with nuxtlabs team and finally approved. Let's start with this and merge this PR!
Future updates will be easier and smoother :slightly_smiling_face: 
Thanks for your contributions guys! :green_heart:

**Comment by smarroufin** [comment]
My bad, there are still requested changes :grimacing: 
@manniL can you approve the PR or eventually commit suggested changes please?

**Comment by manniL** [comment]
Will do ☺️

**Pull request closed without merging**